lms/djangoapps/commerce/utils.py

```diff
@@ -110,20 +112,21 @@ def payment_page_url(self):
 
     def get_add_to_basket_url(self):
         """ Return the URL for the payment page based on the waffle switch.
-
         Example:
             http://localhost/enabled_service_api_path
         """
         if should_redirect_to_commerce_coordinator_checkout():
             return urljoin(settings.COMMERCE_COORDINATOR_URL_ROOT, settings.COORDINATOR_CHECKOUT_REDIRECT_PATH)
         return self.payment_page_url()
 
+    @pluggable_override('OVERRIDE_GET_CHECKOUT_PAGE_URL')
     def get_checkout_page_url(self, *skus, **kwargs):
         """ Construct the URL to the ecommerce checkout page and include products.
 
         Args:
             skus (list): List of SKUs associated with products to be added to basket
             program_uuid (string): The UUID of the program, if applicable
+            course_run_keys (list): The course run keys of the products to be added to basket.
```

Comment:
Isn't this CT-related?

Reply:
If someone from the community want to utilize these methods with another ecommerce system then they would need some sort of identifier. SKU is tightly coupled with legacy e-commerce. The only available identifier here which makes sense is `course_run_key` hence we are adding a note here for the community as well if someone wants to use this functionality